var/spack/repos/builtin/packages/bcl2fastq2/package.py

```diff
@@ -94,7 +93,7 @@ def wrap():
                     tty.msg("cwd sez: {0}".format(os.getcwd()))
                     tarball = glob.glob(join_path('spack-expanded-archive',
                                         'bcl2fastq2*.tar.gz'))[0]
-                    shutil.move(tarball, '.')
+                    install(tarball, '.')
```

Comment:
this is not exactly the same, but I don't think it makes much of a difference. Have you tested this?

Reply:
Haven't tested anything 😄